parm/config/gfs/config.ufs

```diff
@@ -194,19 +193,26 @@ case "${fv3_res}" in
           export tau=6.0
           export rf_cutoff=100.0
           export fv_sg_adj=1800
-          export WRITE_GROUP_GDAS=1
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=10
-          export WRITE_GROUP_GFS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GFS=5
         fi
         ;;
     "C384")
         if [[ "${DO_NEST:-NO}" == "YES" ]] ; then
           export DELTIM=150
-          export layout_x_gdas=8
-          export layout_y_gdas=8
-          export layout_x_gfs=8
-          export layout_y_gfs=8
+          if [[ "${RUN}" = *gdas ]]; then
+            export layout_x=8
+            export layout_y=8
+            export nthreads_fv3=1
+            export nthreads_ufs=1
+            export WRITE_GROUP=2
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=20
+          elif [[ "${RUN}" = *gfs ]]; then
+            export layout_x=8
+            export layout_y=8
+            export nthreads_fv3=2
+            export nthreads_ufs=2
+            export WRITE_GROUP=2
+            export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=20
```

Comment:
`layout_x`, `layout_y`, `WRITE_GROUP`, and `WRTTASK_PER_GROUP_PER_THREAD_PER_TILE` could all be pulled out of this if-elif block.

Reply:
N/M, let's not worry about nesting.